src/WebJobs.Script/Host/ScriptTelemetryClientFactory.cs

```diff
@@ -4,19 +4,23 @@
 using Microsoft.ApplicationInsights;
 using Microsoft.ApplicationInsights.Extensibility.Implementation;
 using Microsoft.ApplicationInsights.WindowsServer.Channel.Implementation;
-using Microsoft.Azure.WebJobs.Host.Loggers;
+using Microsoft.Azure.WebJobs.Logging.ApplicationInsights;
 
 namespace Microsoft.Azure.WebJobs.Script.Host
 {
     /// <summary>
     /// Overrides the default client creation by adding a custom SdkVersion for backend tracking.
     /// </summary>
-    internal class ScriptTelemetryClientFactory : ITelemetryClientFactory
+    internal class ScriptTelemetryClientFactory : DefaultTelemetryClientFactory
     {
```

Comment:
This nuget made it easier to derive from Default -- since this signature changed anyway, I decided to do it here.